### PR TITLE
DPL Analysis: performance improvements

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@Barthelemy is there an issue with one of the QC servers used for tests?

```
17/17 Test  #6: testTaskInterface ................***Timeout  30.07 sec
Running 2 test cases...
ServiceDiscovery: URL using bad/illegal format or missing URL: /v1/agent/service/register
ServiceDiscovery: Response code: 0
initialize
startOfActivity
startOfCycle
endOfCycle
endOfActivity
reset


94% tests passed, 1 tests failed out of 17

Label Time Summary:
slow    =  30.91 sec*proc (2 tests)

Total Test time (real) =  30.09 sec

The following tests FAILED:
	  6 - testTaskInterface (Timeout)
Errors while running CTest
```

**Comment by ktf** [comment]
Code checker issues seems to be related to the new ROOT.

**Comment by Barthelemy** [comment]
@ktf The tests are able to survive without access to the servers (ccdb or consul). I have tested it last Thursday again and they just skip them. There is something else (like a race condition) that happens only on the tests servers. I have made a PR in QC to try and avoid this.

**Comment by ktf** [comment]
Ok, problem with the CI understood. The server hosting the tarballs was restart, and the build areas have an inconsistent state. One more reason to move to S3 also for the tarballs I guess, at least for the CI...

File: Framework/Core/test/benchmark_ASoA.cxx [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    for (auto& track : tracks) {
+      result[i++] = asin(track.snp()) + track.alpha() + M_PI;
+    }
+    benchmark::DoNotOptimize(result);
+  }
+  state.SetBytesProcessed(state.iterations() * state.range(0) * sizeof(float) * 2);
+}
+
+BENCHMARK(BM_ASoADynamicColumnPhi)->Range(8, 8 << 16);
+BENCHMARK_MAIN()
```
Comment (davidrohr) [inline Framework/Core/test/benchmark_ASoA.cxx]: @ktf : `BENCHMARK_MAIN()` is missing the `;` at the end, which breaks the build on my system.
(Don't know why it passed the CI.)

**Pull request merged**